File: app/src/main/java/com/sewageproject/base/MyApplication.java
package com.sewageproject.base;

import android.app.Application;
import android.content.Context;

public class MyApplication extends Application {
    private static Context application;

    public static Context getApplication() {
        return application;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        application = this;
    }
}

File: app/src/main/java/com/sewageproject/base/MyApplication.kt
package com.sewageproject.base

import android.app.Application
import android.content.Context
import com.yechaoa.yutilskt.YUtils.init

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        application = this
        init(this)
    }

    companion object {
        @JvmStatic
        var application: Context? = null
            private set
    }
}